pf4j/src/main/java/org/pf4j/AbstractPluginManager.java

```diff
@@ -49,17 +48,17 @@
 
     private PluginDescriptorFinder pluginDescriptorFinder;
 
-    /*
+    /**
      * A map of plugins this manager is responsible for (the key is the 'pluginId').
      */
     protected Map<String, PluginWrapper> plugins;
 
-    /*
+    /**
```

Comment:
My intention was to use regular block comment for private methods and fields. Private fields won’t have Javadoc generated for them unless we explicitly pass the `-private` option to the Javadoc command.
I am not against to use javadoc comments for private fields and methods but it's good to have a unitary approach throughout the project.

Reply:
Now, I seen your comment frrom commit: 
```
Fix Javadocs for fields so that they are shown in IDE as such
```
It makes perfect sense. It's OK.

Reply:
Yes, the small benefit of these changes is that you can use "Show documentation" feature at the _use_ site to see the documentation in a pop up (Ctrl+Q/F1 depending on the configuration). That's especially useful for `protected` members (and for `private` in a big class).

Shall I submit an issue to update all of them?